src/webview/java/org/medicmobile/webapp/mobile/MedicAndroidJavascript.java

```diff
@@ -365,7 +385,7 @@ private void logException(Exception ex) {
 				.replace("\n", "; ")
 				.replace("\t", " ");
 
-		parent.errorToJsConsole("Execption thrown in JavascriptInterface function: %s", stacktrace);
+		parent.errorToJsConsole("Exception thrown in JavascriptInterface function: %s", stacktrace);
```

Comment:
Nice! I cannot tell you for how long this typo has bugged me! (ranted about it here: https://github.com/medic/cht-core/issues/6162)

Reply:
✨